src/copaw/app/runner/daemon_commands.py

```
@@ -0,0 +1,233 @@
+    parts = ["**Daemon Status**", ""]
+    try:
+        cfg = context.load_config_fn()
+        parts.append("- Config loaded: yes")
+        if getattr(cfg, "agents", None) and getattr(
+            cfg.agents,
+            "running",
+            None,
+        ):
+            max_in = getattr(cfg.agents.running, "max_input_length", "N/A")
+            parts.append(f"- Max input length: {max_in}")
+    except Exception as e:
```

Ruff (0.15.2)

[warning] Do not catch blind exception: `Exception`

(BLE001)